packages/vuetify/src/components/VTreeview/VTreeviewItem.sass

```diff
@@ -20,6 +20,66 @@
     &.v-list--disabled .v-list-item__prepend
       pointer-events: auto
 
+    .v-list-item--slim > .v-list-item__prepend
+      > .v-icon ~ .v-list-item__spacer
+        width: $treeview-item-spacer-width
+
+      &:not(:has(.v-list-item-action))
+        > .v-icon
+          margin-inline-start: $treeview-item-first-icon-margin-start
+
+  .v-treeview-indent-lines
+    position: absolute
+    left: 0
+    height: 100%
+    display: grid
+    padding-left: $treeview-indent-lines-padding-left
+    padding-block: $treeview-indent-lines-gap
+    grid-template-columns: repeat(var(--v-indent-parts, 1), var(--prepend-width))
+    opacity: $treeview-indent-lines-opacity
+    pointer-events: none
+
+  .v-treeview-indent-line
+    &,
+    &::before
+      border: 0px $treeview-indent-line-style $treeview-indent-line-color
+
+    &--leaf,
+    &--line
+      border-left-width: $treeview-indent-line-width
+      height: 100%
+      width: calc(50% + #{$treeview-indent-line-width-half})
```

Comment:
I replaced this with a sass variable, css round() is still too new

Reply:
👍🏼 no problem. Surprisingly Safari has it since '22. It's actually Chrome that lagged and has it for slightly over a year. Weird that it seems totally absent on caniuse.com... or it was grouped in some bundle category. I hope we will be able to use `round(...)` in a couple of years 😃